README.md

```diff
@@ -75,13 +75,12 @@ Channel: #refugerestrooms
 
 Copyright (C) 2014–2017 Teagan Widmer and contributors
 
-This program is free software; you can redistribute and/or modify
+This program is free software. /You can redistribute and/or modify
```

Comment:
Also, this is verbatim from the license: https://www.gnu.org/licenses/gpl-3.0.en.html